lib/spack/spack/cmd/info.py

```diff
@@ -191,6 +194,16 @@ def print_text_info(pkg):
     else:
         print("    None")
 
+    print()
+    print("Tags: ")
```

Comment:
Let's move this to the top, maybe before or after the homepage?  I think we should put more accessible information at the top of `info` and more esoteric stuff (dependencies) at the end.